operators/operator_sdk/osdk-multi-arch-support.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="osdk-multiple-platform-support"]
+= Configuring Operator projects to support multiple architectures and operating systems
+include::_attributes/common-attributes.adoc[]
+:context: osdk-multi-arch
+
+toc::[]
+
+Operator projects that support multiple architectures and operating systems, or _platforms_, can run on more Kubernetes and {product-title} clusters than Operator projects that support only a single platform. Example architectures include `amd64`, `arm64`, `ppc64le`, and `s390x`. Example operating systems include Linux, IBM Z, and Windows.
+
+[NOTE]
+====
+Operator Lifecycle Manager (OLM) defaults to Linux if the operating system is not defined.
+====
+
+As an Operator author, perform the following actions to ensure your Operator project can run on multiple {product-title} platforms:
+
+* Build an image file for every platform you want your Operator to support.
+* Build a manifest list that specifies the platforms that your Operator supports.
+* Set your Operator's node affinity to support multiple architecture compute nodes.
+
```

Comment:
@jaypoulz Do you think we need to further explain the difference between building an Operator that supports installation on multi-archs vs an Operator that supports multi-arch compute nodes?

Reply:
I think it may be worth a callout to docs about multi-arch compute nodes. If I understand correctly, @kelbrown20 helped write those docs, so she might be able to help find a suitable description.